packages/core/src/Stores/notification-store.js

```diff
@@ -441,10 +457,12 @@ export default class NotificationStore extends BaseStore {
     }
 
     @action.bound
-    setClientNotifications(client = {}) {
+    setClientNotifications(client_data = {}) {
+        const { ui, client } = this.root_store;
         const mx_mlt_custom_header = this.custom_notifications.mx_mlt_notification.header();
         const mx_mlt_custom_content = this.custom_notifications.mx_mlt_notification.main();
-        const { ui } = this.root_store;
+        const client_name =
+            client.account_status.authentication.identity.services.onfido.reported_properties.first_name;
```

Comment:
is this real? 😅

Reply:
Yes, this is how the ugly truth looks like 😂

Reply:
I just saw this accidentally
its so error friendly :dagger: 
Isn't it better to use optional chaining for that?
`        const client_name =
            client?.account_status?.authentication?.identity?.services?.onfido?.reported_properties?.first_name;`

Reply:
@ali-hosseini-fs yes, I already added optional chaining in my latest commit yesterday.. We also do not need optional chaining `client.account_status.authentication` as this piece of code will not be executed unless the client is logged in which is already determined from authentication object.